dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/mappers/GraphToTable.kt

```diff
@@ -169,7 +175,15 @@ class GraphToTable {
                                     R.string.line_listing_max_results,
                                     LINE_LISTING_MAX_ROWS,
                                 ),
-                                style = descriptionTextStyle,
+                                style = TextStyle(
+                                    color = Color(0xFF667685),
```

Comment:
don't we have an explicit color for this in the library?. should we create it?

Reply:
It was like that before but it totally makes sense to change it to a color in the library. I'll check if it already exist